assets/platforms/h3c.yaml

```diff
@@ -0,0 +1,38 @@
+---
+platform-type: 'h3c'
```

Comment:
what is this one? can we make it like vendor_osname to be more clear

Reply:
hp_compare, i use short name here. also can be changed

Reply:
ok cool -- yeah lets roll with `hp_compare` then!